google/ggrc-core
# GGRC-4388: Refresh Active Cycles after add

# Issue description
Tree view is not refreshed on 'Active Cycles' tab after adding new Cycle Task to current Workflow.

# Steps to test the changes
1. Create Repeat off workflow
2. Create a task for task group and activate wf
3. At the Active Cycles tab click Create button to create a Cycle task
4. In new cycle task modal fill out the required fields and click Save and Close button
**Actual Result:** Tree view is not refreshed after adding a cycle task 
**Expected Result:** Tree view should be refreshed after adding a cycle task.

# Solution description
handle Cycle Task creation for Active Cycles tab.

# Sanity checklist


# PR Review checklist
